src/main/java/jenkinsci/plugins/influxdb/generators/JenkinsBasePointGenerator.java

```diff
@@ -45,6 +48,11 @@
     public static final String TESTS_SKIPPED = "tests_skipped";
     public static final String TESTS_TOTAL = "tests_total";
 
+    public static final String TOTAL_TIME_IN_QUEUE = "total_time_in_queue";
```

Comment:
is TOTAL_TIME_IN_QUEUE the summation of all subtask queues? If so I would rename to SUMMED_TIME_IN_QUEUE because TOTAL can be misleading.